src/dune/dune_file.ml

```diff
@@ -2045,7 +2045,11 @@ module Coq = struct
        and+ modules = modules_field "modules"
        and+ libraries =
          field "libraries" (repeat (located Lib_name.decode)) ~default:[]
-       and+ theories = field "theories" (repeat Coq_lib_name.decode) ~default:[]
+       and+ theories =
+         field "theories"
+           ( Dune_lang.Syntax.since Stanza.syntax (2, 5)
```

Comment:
Why don't you version it using the version of the coq extension? (`syntax` in this case).

I just noticed that you're using dune's versions for other fields as well. I might be missing something obvious here.

Reply:
The main reason is that we declared "0.1" a moving, unsupported target, but you are right this is an ugly hack.

I wasn't planning to version `coq lang` until we do the first call for "beta testers", the idea was then to set the version to `0.99` , and then if no problem arises declare `1.0` and have regular support.

I think this can happen in 2.5 ; but I'd be happy to bump Coq lang to 0.2 and remove these checks instead (also looking forward to #3270 )

Reply:
Note that after #3270 if we bump Coq to 0.2 we will have to bump dune lang too anyways.